staging/src/k8s.io/apiserver/pkg/server/patch_genericapiserver.go

```diff
@@ -100,3 +91,45 @@ func WithLateConnectionFilter(handler http.Handler) http.Handler {
 		handler.ServeHTTP(w, r)
 	})
 }
+
+// WithNonReadyRequestLogging rejects the request until the process has been ready once.
+func WithNonReadyRequestLogging(handler http.Handler, hasBeenReadyCh <-chan struct{}, s runtime.NegotiatedSerializer) http.Handler {
```

Comment:
s runtime.NegotiatedSerializer not used ?

Reply:
not anymore